lutris/runners/commands/wine.py

```diff
@@ -352,11 +352,10 @@ def wineexec(
     command_parameters = []
     if proton.is_proton_path(wine_path):
         command_parameters.append(proton.get_umu_path())
-        if winetricks_wine and wine_path not in winetricks_wine:
+        if winetricks_wine and wine_path not in winetricks_wine and "winetricks" not in args:
```

Comment:
Not exactly related with the current feature but when debugging, I observed a duplication of the `winetricks` argument. Didn't seem to affect the behavior of Winetricks itself but it may change in the future and trigger a bizarre bug.